azblob/download.go

```diff
@@ -51,6 +53,8 @@ func (azp *Storer) getMetadata(
 	identity string,
 ) (map[string]string, error) {
 
+	azp.log.Debugf("getMetadata: %s", identity)
```

Comment:
See above, my preference in general is to strongly encourage removing all DEBUG, and certainly INFO, telementry from such deep packages

Reply:
removed